src/Components/TextInput/index.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export class TextInput extends React.Component {
+    render() {
+        return (
+            <input onKeyUp={this.props.action} type="text" name={this.props.value} maxLength={this.props.max} minLength={this.props.min} placeholder={this.props.placeholderText} />
```

Comment:
```suggestion
            <input onKeyUp={this.props.action} type="text" name={this.props.value} maxLength={this.props.max} 
            minLength={this.props.min} placeholder={this.props.placeholderText} />
```
very long line - I'd break the line as shown to make it more readable

Reply:
Yep, good idea.